recipes/lyra/all/conanfile.py

```diff
@@ -30,9 +31,13 @@ def source(self):
 
     def package(self):
         self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
-        self.copy("*.h*", dst="include", src=self._source_subfolder+"/include")
+        self.copy("*.h*", dst="include", src=os.path.join(self._source_subfolder, "include"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "lyra")
+        self.cpp_info.set_property("cmake_target_name", "bfg::lyra")
+
+        # TODO: to remove in conan v2 once cmake_find_package* generators removed
```

Comment:
```suggestion
        # TODO: to remove in conan v2 once cmake_find_package* generators removed
        self.cpp_info.components["_lyra"].set_property("cmake_target_name", "bfg::lyra")
```

Reply:
🤔 Why is it better to add a component? AFAIK this project only has one part for I thought the "global space" was better.

Thanks in advance for helping me along

Reply:
there is a component due to cmake_find_package, it avoids to declare un unwanted target.